lib/LaTeXML/Package/siunitx.sty.ltxml

```diff
@@ -24,9 +24,10 @@ use LaTeXML::Package;
 #  * table alignments!
 #%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 #======================================================================
+# recent siunitx have an expectation for latex3 capability.
+RequirePackage('expl3');
```

Comment:
I'd hoped to avoid this, since straightforward use of siunitx should work without. However, the loading of expl3 may indeed have been expected, so...